rules/python/lang/insecure_cookie.yml

```diff
@@ -0,0 +1,40 @@
+patterns:
+  - pattern: |
+      $<COOKIE_INIT>[$<_>]['secure'] = $<FALSE>
```

Comment:
It might be worth splitting this (ie. putting `$<COOKIE_INIT>[$<_>]` in it's own rule), to allow it to match when it's not just a single expression